egs/yomdle_fa/v1/local/normalized_scoring/normalized_scoring.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+# This script provides normalized scoring for the various slam dataset languages.
```

Comment:
Could you please show me where this script is used in the recipe?

Reply:
Everything in the normalized_scoring dir was specifically to use the same normalization as Stephen. It's separate from the Kaldi stuff not actually used in the scripts if ./run_end2end.sh is run. I can remove the entire directory if need be.

Reply:
we can also, run the normalize scoring without ref_files. ref_files are the test utterances. in the normalize scoring script we can replace the path to ref_file with a path to data/test/text.

Reply:
Usually it's around 3-4% WER improvement. I removed the normalization that was done in the ./local/wer_output_filter. I can put it back in to remove the reliance on an external scoring script. This was mostly added to ensure that over the summer everyone was normalizing in the same way. Some of the stuff are things like "farsi vs arabic yeh" and "tatweel".

Reply:
I guess we can keep them and you can put them back in wer_output_filter.

Reply:
okay, I've added them back